### PR TITLE
Director-free TAG

### PR DESCRIPTION
**Pull request by frivoal**
This lets the TAG pick its own chair and replaces the Director appointment with appointment by a new TAG Appointment Committee (and does a few minor related cleanups).

***

### :boom: Error: 503 Service Unavailable :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Sep 12, 2022, 5:47 AM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [HTML Diff Service](http://services.w3.org/htmldiff) - The HTML Diff Service is used to create HTML diffs of the spec changes suggested in a pull request.

:link: [Related URL](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fpr-preview.s3.amazonaws.com%2Fw3c%2Fw3process%2Fpull%2F611%2F0c29dc9.html&doc2=https%3A%2F%2Fpr-preview.s3.amazonaws.com%2Ffrivoal%2Fw3process%2Fpull%2F611.html)

```
<html><body><h1>503 Service Unavailable</h1>
No server is available to handle this request.
</body></html>

```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/w3process%23611.)._
</details>

### CODE REVIEW

**Comment by torgo** [comment]
Just discussing this in today's TAG f2f. Even though this proposal has come out of a TAG session originally, we think it can be improved. 

Firstly, yes the TAG agrees that we should be able to pick our own chair(s).

Thinking of the purpose of appointments, they are intended to fill a gap - in expertise, background,  experience, or demographics - that may exist after an election. There is also a value to having continuity - which is arguably what the appointed TAG members have more been used for historically. On the flip side, continuity should be balanced by fresh perspectives as needed. We strongly feel this needs to be preserved.

We think it might be more effective to formalize that it's the Team that appoints the seat but that the Team is charged with reaching out to the W3C community (including the TAG), to come to these decisions on appointments. The Team will have a mandate to seek input from an explicit set of constituencies, and weigh it, in order to make a decision.

This saves the need to convene a large group that potentially changes each year, who then have to spend time meeting and seeking consensus. It also reduces the risk of the nomination process becoming a shadow election, if the nomination committee is composed primarily of member representatives.

Further to this, we should explore the idea that appointed members put themselves forward for election during the election periods that coincide with their appointments expiring.  This will encourage appointed members to go through the same process as elected members on the TAG.

In summary, we think more discussion is needed before we make a process change here. Maybe we can discuss at TPAC and come to a clearer consensus?

**Comment by fantasai** [comment]
The Process CG discussed this on the call today.

- We opened https://github.com/w3c/w3process/issues/613 to discuss the continuity of TAG appointments.
- We are requesting the TAG and the AB to arrange a joint session at TPAC to discuss what approach they want to take and instruct the Process CG appropriately.

**Comment by dwsinger** [comment]
> We think it might be more effective to formalize that it's the Team that appoints the seat but that the Team is charged with reaching out to the W3C community (including the TAG), to come to these decisions on appointments. The Team will have a mandate to seek input from an explicit set of constituencies, and weigh it, in order to make a decision.
> 
> Further to this, we should explore the idea that appointed members put themselves forward for election during the election periods that coincide with their appointments expiring. This will encourage appointed members to go through the same process as elected members on the TAG.

At the August F2F the AB had the same sense; that we can get much of the effect of the TAG appointment committee by asking the team to consult, but avoid a lot of the complexity. And that term limits, and standing for election, may be a good way to ensure that we really are balancing the TAG for diversity, including skill, knowledge, perspective, etc.

Something like
* * * *

TAG appointments are made by the team, after consulting with the chairs and the members of the TAG not involved in the concurrent election. TAG appointments should be used to balance diversity, including diversity of technical background, knowledge, and skill sets.

A person may hold at most two consecutive appointed terms. To be eligible for a second consecutive appointed term, the candidate must have stood in the election for that term. There is no limit to the number of non-consecutive terms a person may be appointed to.

* * * 

How much do we need to describe the expected consultation?

**Comment by frivoal** [comment]
The proposal adopted by the AB was that, similarly to the [choice of AB chairs](https://www.w3.org/2021/Process-20211102/#ABParticipation), the Team would consult with the TAG and propose appointees, but that these would have to be ratified by the TAG, not just that the Team alone could appoint. (In this context, “by the TAG” means mid-term members + newly elected members).


As for your second proposed paragraph, I think this is orthogonal to the question of who does the appointing, and is being handled in #613.

**Comment by dwsinger** [comment]
agreed, I forgot TAG ratification, and yes, term limits are separately tracked

**Comment by dwsinger** [comment]
Trying again. Keeping the term limits here so we can see an integrated text. Do we need to explain who gets to vote in the confirmation ballot, in any more detail than this?

* * * * *

The Team is responsible for nominating for appointment 3 of the participants to the [Technical Architecture Group](https://www.w3.org/Consortium/Process/Drafts/director-free/#technical-architecture-group). This mechanism complements the election process. The nominations are formally made after the results of the election are known, and are are confirmed by secret ballot of the newly-formed TAG (after election), by majority vote.

The Team should use appointments to support a diverse and well-balanced [TAG](https://www.w3.org/Consortium/Process/Drafts/director-free/#technical-architecture-group). The Team should consult with the participants of the current TAG whose terms are not expiring (elected or appointed), and with the Chairs of Working Groups, before and during the election period.

A person may hold at most two consecutive appointed terms. To be eligible for a second consecutive appointed term, the candidate must have stood in the election for that term. There is no limit to the number of non-consecutive terms a person may be appointed to.

**Comment by frivoal** [comment]
@dwsinger I do think we need a little bit more than that, PR incoming, will share soon.

**Comment by frivoal** [comment]
Pull request on pull requests are hard to do, so I updated the existing pull request by adding a commit. You can:
* look at this whole pull request to see what this would add to the Process in general to handle the director-free TAG, or 
* just look at https://github.com/w3c/w3process/pull/611/commits/a63bb758ff41bc23d42d13c5be28ee8755a0f478 to see just the difference between TAG appointment committee and Team-appoints-TAG-ratifies (the diff is a little hard to read, as it is removing a lot), or
* just look at https://github.com/w3c/w3process/pull/611/commits/e841f1367fef7fbf48e5d7f400504ce9fb2ec152 for just the addition of term limits.

File: index.bs [hunk 1]
```diff
@@ -990,32 +990,26 @@ Composition of the Technical Architecture Group</h5>
...
-	who <em class="rfc2119">must</em> be one of the participants.
-	The team also appoints a <a href="https://www.w3.org/Guide/teamcontact/role.html">Team Contact</a> [[TEAM-CONTACT]] for the TAG,
+	Participants in the [=TAG=] choose by consensus their Chair or co-Chairs;
+	in the absence of consensus, the [=Team=] appoints the Chair or co-Chairs of the TAG.
+	The Chair or co-Chairs <em class="rfc2119">must</em> be selected from the participants of the TAG.
+	The [=Team=] also appoints a <a href="https://www.w3.org/Guide/teamcontact/role.html">Team Contact</a> [[TEAM-CONTACT]] for the TAG,
 	as described in <a href="#ReqsAllGroups"></a>.
 
-	The terms of elected and Director-appointed TAG participants are for <span class="time-interval">two years</span>.
+	The terms of TAG participants are for <span class="time-interval">two years</span>.
```
Comment (TallTed) [inline index.bs:1006]:
```suggestion
	The terms of TAG participants last for <span class="time-interval">two years</span>.
```

File: index.bs [hunk 2]
```diff
@@ -990,32 +990,26 @@ Composition of the Technical Architecture Group</h5>
...
 	Terms are staggered so that each year three elected terms,
 	and either one or two appointed terms expire.
```
Comment (TallTed) [inline index.bs:1008]:
```suggestion
	and either one or two appointed terms, expire.
```

File: index.bs [hunk 3]
```diff
@@ -1047,7 +1041,7 @@ Participation in Elected Groups</h4>
 Expectations for Elected Groups Participants</h5>
 
 	[=Advisory Board=] and [=TAG=] participants have a special role within W3C:
-	they are elected by the Membership and appointed by the Director
+	they are elected by the Membership and appointed
```
Comment (TallTed) [inline index.bs:1044]:
```suggestion
	they are elected by the Membership and appointed by the [=Team=]
```

File: index.bs (hunk 2 above)
Comment (frivoal) [reply index.bs:1008]: I think I'd rather leave out both commas (the one you're proposing to add and the one after "three elected terms") than just add that one. Possibly adding a comma after "so that each year". What do you think?

File: index.bs [hunk 4]
```diff
@@ -1213,6 +1209,52 @@ Advisory Board and Technical Architecture Group Elections</h5>
...
+	is subject to ratification by secret ballot
+	by two thirds of the [=TAG=].
+	In the case of regularly scheduled election,
+	ratification is by the [=TAG=] members of the upcoming term.
+
+	For regularly scheduled elections,
+	selection begins once the results of the elections are known,
+	and the [=Team=] <em class=rfc2119>should</em> announce the ratified appointment(s)
+	no later than the start of the regularly scheduled term.
+	When an appointed seat is vacated out of a regularly scheduled election,
```
Comment (TallTed) [inline index.bs:1252]:
```suggestion
	When an appointed seat is vacated outside of a regularly scheduled election,
```

File: index.bs (hunk 2 above)
Comment (TallTed) [reply index.bs:1008]: See new suggestion, below...

Comment (TallTed) [inline index.bs:1008]: A bit of phrase shuffling makes me feel better about dropping both commas.
```suggestion
	Terms are staggered so that three elected terms
	and either one or two appointed terms expire each year.
```

Comment (frivoal) [reply index.bs]: thanks, took the new suggestion instead.

File: index.bs [hunk 5]
```diff
@@ -1213,6 +1209,52 @@ Advisory Board and Technical Architecture Group Elections</h5>
...
+
+	Note: Individuals who have reached the limit of two consecutive appointed terms
+	<em class=rfc2119>may</em> freely run for election
+	if they wish to continue serving on the [=TAG=].
+
+	The [=Team=]'s choice of appointee(s)
+	is subject to ratification by secret ballot
+	by two thirds of the [=TAG=].
+	In the case of regularly scheduled election,
+	ratification is by the [=TAG=] members of the upcoming term.
```
Comment (mnot) [inline index.bs:1246]: Should it be explicit that ratification takes place_before_ announcement?

File: index.bs [hunk 6]
```diff
@@ -1213,6 +1209,52 @@ Advisory Board and Technical Architecture Group Elections</h5>
 	Refer to <a href="https://www.w3.org/2002/10/election-howto">How to Organize an Advisory Board or TAG election</a> [[ELECTION-HOWTO]]
 	for more details.
 
+<h5 id="TAG-appointments">
+Technical Architecture Group Appointments</h5>
+
+	The [=Team=] is responsible for appointing
+	3 of the participants to the [=Technical Architecture Group=].
```
Comment (mnot) [inline index.bs:1216]: Does the Team _have_ to apoint three participants?

File: index.bs (hunk 5 above)
Comment (frivoal) [reply index.bs:1246]: I think it is already? In the next paragraph: “the Team should announce the *ratified* appointment […]”

File: index.bs (hunk 6 above)
Comment (frivoal) [reply index.bs:1216]: Good question.
I think they ought to try, but they could fail to find the right number of suitable people, or, possibly more importantly, they could fail to get them all ratified… So I don't think we can strictly mandate it. I think the current wording works. It combines:
* the fact that the Team is responsible for getting 3
* the fact that vacant seats *should* be filled
* the fact that the team *should* get their appointment ratified before the start of the term

File: index.bs (hunk 1 above)
Comment (TallTed) [reply index.bs:1006]: Or even
```suggestion
	Each TAG participant's term lasts for <span class="time-interval">two years</span>.
```

**Comment by fantasai** [comment]
We are currently reviewing this text, and barring any objections will merge it in at the next Process CG call.

File: index.bs [hunk 7]
```diff
@@ -990,32 +990,26 @@ Composition of the Technical Architecture Group</h5>
...
 
 		<li>
 			Six participants elected by the [=Advisory Committee=]
 			following the <a href="#AB-TAG-elections">AB/TAG nomination and election process</a>.
 	</ul>
 
-	The [=Team=] appoints the Chair of the TAG,
-	who <em class="rfc2119">must</em> be one of the participants.
-	The team also appoints a <a href="https://www.w3.org/Guide/teamcontact/role.html">Team Contact</a> [[TEAM-CONTACT]] for the TAG,
+	Participants in the [=TAG=] choose by consensus their Chair or co-Chairs;
```
Comment (TallTed) [inline index.bs:1000]:
```suggestion
	Participants in the [=TAG=] choose their Chair or co-Chairs by consensus;
```
